src/config/texts.js

```diff
@@ -70,15 +70,20 @@ export const texts = {
   },
   homeTitles: {
     about: 'Über die App',
+    company: 'Städtische Unternehmen',
     events: 'Veranstaltungen',
+    lunch: 'Mittagstisch',
```

Comment:
`lunch` is not used anywhere

Reply:
removed in 3795523c9fe7f462df6678bc1c684c0c58969914